tests/foreman/api/test_role.py

```diff
@@ -1238,6 +1238,7 @@ def test_positive_taxonomies_control_to_superadmin_without_org_admin(
         entities.User(id=user.id).delete()
         with pytest.raises(HTTPError):
             user_role.read()
+        with pytest.raises(HTTPError):
```

Comment:
This looks worse to me than just having 2 calls in the same raises block

Reply:
which is more functionally correct for the test, though?
shouldn't both raise HTTPErrors?
what if the `user_role.read()` doesn't raise an HTTPError but `user.read()` does?

Reply:
You are definitely right, this is more correct. I'm assuming the second statement here hasn't ever executed, actually, since the first statement will always raise an error ( it's been deleted ) and you'd leave the ocntext